### DISCUSSION TITLE
Remove the YOLO "achievement"

### DISCUSSION THREAD

**Discussion by nickva** [community/community#18590] [Other Feature Feedback, Questions, & Ideas] [answered]
It looks silly and can be taken the wrong way. Not all reviews are through Github. For instance, Apache projects often have voting and decision making on the mailing lists. Github isn't aware of that and assigns "YOLO" achievements for them. I'd rather not have that on my profile if possible.

**Comment by ghost** [comment]
YOLO

**Reply by afkvido** [reply]
NOBODY REMOVE THIS ACHIVEMENT IT IS NICE

**Reply by ibidathoillah** [reply]
We need more achievement, and so people will encouraged to try any possibility what can he do on github.

**Reply by klardotsh** [reply]
> We need more achievement, and so people will encouraged to try any possibility what can he do on github.

GitHub isn't a game, and it shouldn't be trying to be a social network either, it's a tool I use to ostensibly get work done. If I want to learn what the possibilities of the platform are, give me good docs and I'll read them when and if I care to know what new possibilities exist.

**Reply by ghost** [reply]
That's not a really good idea to keep this YOLO badge, gives it bad impressions I think

**Reply by eglove** [reply]
> GitHub isn't a game

Shhhhhhhh, just YOLO

**Comment by AlexWinder** [comment]
You can remove achievements entirely from your profile at https://github.com/settings/profile

![image](https://user-images.githubusercontent.com/24867605/173617127-ed65e3fa-210e-46d1-8001-876762a1cbf2.png)

**Reply by nickva** [reply]
I like achievements, it's a nice touch. I just think they have to make sense. In this case the YOLO one doesn't make sense as the decision in the particular PR it referenced had been made on a mailing list.

**Reply by manhhung-fpt** [reply]
thanks for the information

**Reply by Albert481** [reply]
Great info!

**Reply by antonagestam** [reply]
Nice, for me as a professional working with code, these achievements come off as borderline patronizing. Gamifying version control in this way doesn't strike me as a good idea.

**Reply by rahmatsubandi** [reply]
Hi, where can I get YOLO?

**Comment by NowDev** [comment]
Turning off specific achievements would be a nice feat, but for now you can only turn off everything.

**Reply by batuhan-bas** [reply]
Yes, this would be great...

**Comment by JonathanSchndr** [comment]
Sorry, but the YOLO achievement is gold. We need more of them. It would be important that everyone can hide every achievement.

**Reply by MichaelWheldon** [reply]
I think you mean "Soz bro lmao yeetzor xD". Go all in my guy, we live in meme land now.

**Comment by Moon-developer** [comment, marked as answer]
This Achievement is embarrassing without context to the situation and might be taken wrongly by hiring members when viewing.

**Reply by fdwr** [reply]
Evidently you can even get it for PR's that just integrate into feature branches too (not master), even if all the respective commits within that RI have been peer reviewed. 🤷‍♂️

**Reply by minhazahmed20** [reply]
Please help me recover method and solve the problem

**Reply by ShubhamJagtap2000** [reply]
> This Achievement is embarrassing without context to the situation and might be taken wrongly by hiring members when viewing.

Can you please help me in removing this achievement...

**Reply by ThomasThelen** [reply]
Do you really want to work for someone that judges you based your GitHub achievements?

**Reply by ShubhamJagtap2000** [reply]
Naah, no one has done it with me and I haven't hidden the badge not gonna do it... thank you.

**Reply by dolanmiu** [reply]
@Moon-developer Your hiring manager and interviewers are supposed to judge you based on your interview performance and past experience, not on some light-hearted badge. Chill out.

>  without context to the situation

Yes you're right, no context to the situation, so why would they care? You are taking this way too seriously

**Reply by dvklopfenstein** [reply]
I use pull requests because GItHub includes PR titles as bullet points whenever I create a new release to show researchers what changed.

I like that feature a lot, so started creating pull requests.

But if there is nobody to approve my pull requests but me, GitHub "awards" me with a YoLo "achievement. Not cool.

**Reply by hughesjs** [reply]
If someone is making a hiring decision based on what Github achievements you have... You probably don't want to be working for them...

Also, I'm fairly sure you'd get this badge for merging a PR on one of your own repos where you're the only contributor... What you supposed to do, review your own PR?

**Reply by eglove** [reply]
The paranoia, the fear! This whole thread has been a pretty funny ride.

**Reply by rilysh** [reply]
It's not necessary to have a GitHub account to get hired for a job related in "programming" field. Most of the time, you have apply by yourself by proposing a project you've done. It doesn't matter where you're going to host it, they just would like to know what you're capable to.

Companies shouldn't judge by GitHub account, anyway.

**Comment by jafludo** [comment]
Like that achievement !

**Comment by adamdyer** [comment]
Just got this award on a PR that had been reviewed seconds before so it's also got a bug in it which is fitting.

**Comment by giulianoojeda** [comment]
this award is bugged indeed.. lol don't know if it's appropiate anyways

**Comment by pyramation** [comment]
I appreciated the achievements that seemed to be "real" and professional, like the arctic code vault. These types of achievements take away from the real ones.

**Comment by rilysh** [comment]
That's not the point. If anyone hire a member who have a GitHub account and also have a YOLO badge will not disagree their skills. Consider, everyone would know that, YOLO badge was given by GitHub not a dumb achievement or mark that point something silly. When hiring someone, staff obviously would know what that badge means.

Though, I agree that this is quite unprofessional word "YOLO" compared to "Arctic Code Valut". At this point if its really annoying, only way to get rid by disable.

**Comment by Lumpish-Haggard** [comment]
It is already a very buggy achievement. I mean getting YOLO as an achievement is an achievement in itself. But I do see why some people might find it patronizing and a selective showcase of Achievement should become reality sooner rather than later.

**Comment by attwice** [comment]
It's nice feature.

**Reply by wg-d** [reply]
Yes, I like that too.

**Comment by 45739847** [comment]
No need to remove, just make it less "silly"

**Comment by Y0SH4** [comment]
i think is good archievement hahaha

**Comment by vsoch** [comment]
When this is something that more people have done than not, is it really an achievement?

I love the new ideas, but I think I'd want minimally an ability to hide specific ones that I don't want to show in my profile, and not "all or nothing" as it is now. The arctic code vault one, for example, is really cool, but the YOLO could come across as silly, or even harmful if someone's profile is being assessed as a candidate.

**Reply by StingyJack** [reply]
> When this is something that more people have done than not, is it really an achievement?

Is learning to read or write not an achievement?

**Comment by ZehMatt** [comment]
Pretty stupid badge, I have some projects that I work on alone mostly and I've enough tests to be confident about what I do so I create a PR and wait for the CI to be all green and then merge it and I get an awkward achievement for this?

**Comment by StingyJack** [comment]
This badge should be applied to someone who gives code a passing review but did not look at any of the files and made no other comment. 

And it should be sized according to how many files/changes were in the PR.  More than 50 files or changes should change the person's avatar/profile icon for a week (or until they earn it back by doing a proper review instead of a "rubber stamp").

**Comment by vibecastor** [comment]
I'd prefer that this particular "achievement" not be listed or reconsidered.  Taken out of context it looks bad and is frankly kind of  insulting as a Github user.  Did the product owners consider how this might read if say your future employer happened to see it on your profile?  Did the product owners consider all use cases like teams/developers that might have already reviewed code by other means outside the Github ecosystem?  I'm fully in support of good code review but not everybody is lucky enough to work on a team with a formal process and not everybody is merging PR's for some fancy open source project like React.

**Reply by ThomasThelen** [reply]
I think that the larger problem is judging the quality of an engineer/self based on achievements on a social programming platform.

**Reply by eglove** [reply]
Yes, it's like being ashamed of a contribution graph that isn't full. Calm down, write a script to draw cats on it, and enjoy GitHub.

**Comment by AngelGzS** [comment]
In my case, I'm a solo developer contributing to some private repositories. It's embarrassing to have that badge on my profile, and there should be an option to hide that badge, lol. BTW I like all the other badges :)

**Reply by eglove** [reply]
YOLO is good. YOLO is pride.

**Comment by coreymtaylor** [comment]
check in https://github.com/settings/profile

**Comment by aaronlamz** [comment]
try this https://github.com/settings/profile
<img width="363" alt="image" src="https://user-images.githubusercontent.com/3964466/179009756-9c6d125a-8ee2-410a-b02c-3505b61f3ce1.png">

**Reply by StingyJack** [reply]
"achievements" going YOLO is not an achievement

**Reply by ThomasThelen** [reply]
What, you've never had a Four Loko?

**Comment by eglove** [comment]
Don't be ashamed. EMBRACE THE YOLO!

**Comment by dolanmiu** [comment]
How do I obtain this achivement?? I want it lol

I commit to master all the time without review (for personal/small projects)

**Reply by eglove** [reply]
Commit to a seperate branch on a repo with CI. (Sonar Cloud, Snyk, tests, any GitHub actions that takes a few mins), merge it despite a fail, or before it finishes.

**Comment by jfullstackdev** [comment]
latest update, you can now pick what you want to hide, unlike in the past, it was all or nothing

go to Achievements Tab, click the `Achievements` word, it is a link

![image](https://user-images.githubusercontent.com/47092464/204711684-e7c5ebb6-57f9-4fbf-b8fe-96c2182f41bb.png)

or this URL, it's same thing 

https://github.com/[username]?tab=achievements

you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711801-dde96c4d-5133-4201-ad8a-9b23b55ed5d2.png)

then, click any badge, hover left then you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711940-2c5e8b4d-7580-4145-afc0-268f1739d8de.png)

then boom ... you now have the option to hide those badges that you want to hide

**Reply by AngelGzS** [reply]
Thank you so much!
